File: Server/src/com/mytalk/server/logic/shared/modelClient/PersonalData.java
/**
* Filename: PersonalData.java
* Package: com.mytalk.server.logic.shared.modelclient
* Author: Nicolò Toso
* Date: 2013-04-18
*
* Diary:
*
* Version |
Date
| Version | Date       | Developer | Changes
* --------+------------+-----------+------------------
* 0.1	  |	2013-04-12 | NT        | [+] Creazione classe e metodi   
*
* This software is distributed under GNU/GPL 2.0.
*
* Software licensed to:
* - Zucchetti SRL
*/

package com.mytalk.server.logic.shared.modelClient;

public class PersonalData{
		
	private String username;
	private String password;
	private String name;
	private String surname;
	private String email;
	private String ip;

	public PersonalData(){}
	
	public PersonalData(String us, String pwd, String n, String s,String mail, String i){
		username=us;
		password=pwd;
		email=mail;
		name=n;
		surname=s;
		ip=i;
	}
	
	public String getUsername(){
		return username;
	}
	
	public void setUsername(String u){
		username=u;
	}
	
	public String getPassword(){
		return password;
	}
	
	public void setPassword(String p){
		password=p;
	}
	
	public String getName(){
		return name;
	}
	
	public void setName(String n){
		name=n;
	}
	
	public String getSurname(){
		return surname;
	}
	
	public void setSurname(String s){
		surname=s;
	}
	
	public String getEmail(){
		return email;
	}
	
	public void setEmail(String e){
		email=e;
	}
	
	public String getIp(){
		return ip;
	}
	
	public void setIp(String i){
		ip=i;
	}
}

File: Client/com/mytalk/client/model/modelstruct/PersonalData.js
/**
* Filename: PersonalData.js
* Package: com.mytalk.client.model.modelstruct
* Dependencies: 
* Author: Agostinetto Mattia
* Date: 2013-04-20
*
* Diary:
*
| Version | Date       | Developer | Changes
* --------+------------+-----------+------------------
* 0.1     | 2013-04-20 | MA        | [+] Scrittura classe del Model
*
* This software is distributed under GNU/GPL 2.0.
*
* Software licensed to:
* - Zucchetti SRL
*
* Descrizione della _classe_
*/

MyTalk.PersonalData = DS.Model.extend({
  userId: DS.attr('number'),
  username: DS.attr('string'),
  password: DS.attr('string'),
  name: DS.attr('string'),
  surname: DS.attr('string'),
  email: DS.attr('string'),
  md5: DS.attr('string'),
  ip: DS.attr('string'),

  avatarTiny: function() {
    return 'http://www.gravatar.com/avatar/' + this.get('md5') + '?s=48&d=identicon';
  }.property('md5'),

});

/**
* Descrizione di userId
* 
* @property -userId
* @type {number}
*
*/

/**
* Descrizione di username
* 
* @property -username
* @type {string}
*
*/

/**
* Descrizione di password
* 
* @property -password
* @type {string}
*
*/

/**
* Descrizione di name
* 
* @property -name
* @type {string}
*
*/

/**
* Descrizione di surname
* 
* @property -surname
* @type {string}
*
*/

/**
* Descrizione di email
* 
* @property -email
* @type {string}
*
*/

/**
* Descrizione di md5
* 
* @property -md5
* @type {string}
*
*/

/**
* Descrizione di ip
* 
* @property -ip
* @type {string}
*
*/

/**
* Descrizione di avatarTiny
* 
* @property -avatarTiny
* @return {string}
*
*/